lib/helper/Puppeteer.js

```diff
@@ -13,7 +13,6 @@ const {
   ucfirst,
   fileExists,
   chunkArray,
-  toCamelCase,
```

Comment:
I thought this was in this PR https://github.com/Codeception/CodeceptJS/pull/2059/files#diff-0fff3879242c78dedc81559a6091f25bR16. Any reason for removing this?

Reply:
That's interesting, I did not remove or touch this line. I will try to redo a better PR.

Reply:
Yes, this was a part of a previous fix.